modules/dcache/src/main/java/diskCacheV111/services/PoolStatisticsV0.java

```diff
@@ -241,13 +183,14 @@ protected SimpleDateFormat initialValue()
 
     private final CellNucleus      _nucleus;
 
-    private CellCron.TimerTask _hourly;
+    private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(1);
+    private final Runnable hourlyRunner = new HourlyRunner();
```

Comment:
Since you only use the hourlyRunner object in a single place, there is no need to store it globally.